src/ui/Editor.ts

```diff
@@ -31,7 +31,7 @@ export function Editor() {
           onChange: (value) =>
             set({
               vehicleConfig: { ...get().vehicleConfig, radius: value },
-              raycast: { ...get().raycast, wheelInfos: get().raycast.wheelInfos.map((info) => ({ ...info, radius: value })) },
+              raycast: { ...get().raycast, wheelInfos: get().raycast.wheelInfos.map((info) => ({ ...info, radius: value })) as WheelInfos },
```

Comment:
let's figure out how to not cast here.

Reply:
Yeah, That's going to be a funny one. As Typescript cannot be sure that raycast.wheelInfos returns exact an array with 4 elements. We could switch to a custom type that ensures this.

Reply:
Okay so the `.map()` is a bit wrong here. It will convert a tuple definition `[number, number]` to `number[]` and that doesn't match and typescript correctly will error that. It's fixed now by not using a tuple defintion until I can get a workaround or typescript will play nice.